dev-env/testsvc_ipv6.yaml

```diff
@@ -0,0 +1,37 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
```

Comment:
Can you change the commit message explaining why we need this change and the fact that is frr only?